### DISCUSSION TITLE
tiktok的规则要咋写

### DISCUSSION THREAD

**Discussion by sanwuping** [kkkgo/PaoPaoDNS#219] [问答和帮助]
我自己找了几个规则写进了force_forward_list.txt，电脑上面没有啥问题，可以观看。但是苹果手机无法使用。
如果把苹果手机的默认网关改为clash网关，DNS用PAOPAO，手机上面也可以看，要是网关改为默认路由器网关，就不可以。

**Comment by biggbuddy** [comment]
可否分享一下你的tiktok规则?

**Comment by kkkgo** [comment]
参考 https://github.com/kkkgo/PaoPaoGateWay/discussions/98
tiktok可能使用了内置硬编码的DNS服务器，你在路由上drop掉比如8.8.8.8 8.8.4.4，此外，可能还需要把dns.google.com和dns.google加入force_dnscrypt_list。
